bitnami/nginx/templates/servicemonitor.yaml

```diff
@@ -31,11 +31,11 @@ spec:
       {{- end }}
       {{- if .Values.metrics.podMonitor.relabelings }}
       relabelings:
-      {{- include "common.tplvalues.render" (dict "value" .Values.metrics.podMonitor.relabelings "context" $) | nindent 8 }}
+      {{- include "common.tplvalues.render" (dict "value" .Values.metrics.serviceMonitor.relabelings "context" $) | nindent 8 }}
       {{- end }}
       {{- if .Values.metrics.podMonitor.relabelings }}
```

Comment:
Shouldn't these if statements be modified to be `serviceMonitor` as well?

Reply:
yes, you are right, I missed that, would you like to create the PR fixing it?